Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+@Library('sec_ci_libs@v2-latest') _
+
+def master_branches = ["master", ] as String[]
+def paths = []
+
+// using mesos node because it's a lightweight alpine docker image instead of full VM
+node('mesos') {
+  stage("Verify author") {
+    user_is_authorized(master_branches, '8b793652-f26a-422f-a9ba-0d1e47eb9d89', '#tools-notify')
+  }
+}
+
+node('mesos-ubuntu') {
+  def shcmd = { String command ->
```

Comment:
If this is move to line 7 or 8, this would then be available to the parallel builders.

Reply:
@colin-msphere  believe me I tried. Small details like this are what made this take so long. Jenkinsfile is very picky about what is outside of a "node" scope. Especially if that external code is being called by something INSIDE a "node" scope